cell_classification/segmentation_data_prep.py

```diff
@@ -58,17 +59,30 @@ def __init__(
             cell_mask_key (str):
                 The key in the data_folder that contains the cell mask labels
         """
-        pass
-        if normalization_dict_path is not None:
-            self.normalization_dict = json.load(open(normalization_dict_path, "r"))
+        os.makedirs(tf_record_path, exist_ok=True)
+        # assign selected markers
+        self.conversion_matrix = pd.read_csv(conversion_matrix_path)
+        if selected_markers is None:
+            self.selected_markers = list(self.conversion_matrix.columns)
+            if cell_type_key in self.selected_markers:
```

Comment:
This is fine for now, once we have a dedicated input validation function we can move that logic there instead

Reply:
I moved loading and calculating the `normalization_matrix` from `__init__` to `make_tf_record` function and added `test_make_tf_record` accordingly.